src/gui/torrentcontentfiltermodel.cpp

```diff
@@ -111,16 +111,14 @@ void TorrentContentFilterModel::selectAll()
 {
     for (int i = 0; i < rowCount(); ++i)
         setData(index(i, 0), Qt::Checked, Qt::CheckStateRole);
-
-    emit dataChanged(index(0, 0), index((rowCount() - 1), (columnCount() - 1)));
+    emitModelUpdate(index(0, 0));
```

Comment:
I wouldn't use `emit` prefix in this function name.

Reply:
Honestly I'm confusing by implementation of this filter model. Shouldn't it just bypass input data to the handled model and bypass its signals back to the outside world?

Reply:
>I wouldn't use emit prefix in this function name.

Renamed.

>Honestly I'm confusing by implementation of this filter model. Shouldn't it just bypass input data to the handled model and bypass its signals back to the outside world?

Aren't they are different models? One is the original and one is filtered.

Reply:
>Aren't they are different models? One is the original and one is filtered.

Original (source) model actually handles the data. Filtering model is just a proxy on top of it. When you set any data to proxy model it bypass this data to source model. Also it translates `dataChanged()` signal from source model and emits the corresponding `dataChanged()` signal itself.

Reply:
>Original (source) model actually handles the data. Filtering model is just a proxy on top of it. When you set any data to proxy model it bypass this data to source model. Also it translates dataChanged() signal from source model and emits the corresponding dataChanged() signal itself.

I pushed an update. Maybe it addresses this concern more or less.